docs/content/management-bundles.md

```diff
@@ -711,34 +711,6 @@ bundles:
 
 **NOTE:** the S3 `url` is the bucket's regional endpoint.
 
-##### Credential Provider Chaining
```

Comment:
Looks like another rebase/merge mishap. LMK if you could use some assistance, git can be hard. 😅 But it shouldn't be what's eating your time.